src/js/map/boundary_type_box.js

```diff
@@ -0,0 +1,161 @@
+import {Observable} from "../utils";
+
+let selectElement = $('.map-geo-select')[0];
```

Comment:
why did you opt for using them outside the class? and not inside it?

Reply:
this was the way the project was coded before I started working on this project so I continued the habits. should I declare them in class and use with `this. `?